Makefile

```diff
@@ -223,6 +224,12 @@ ifneq ($(IN_DOCKER),)
 endif
 	docker exec -t -i ${DOCKER_NAME} ./manage.py shell
 
+dbshell:
+ifneq ($(IN_DOCKER),)
+	$(warning Command is designed to be run in the host)
+endif
+	docker exec -t -i ${DOCKER_NAME} ./manage.py dbshell
+
```

Comment:
Not really part of this pull request but small enough so okay. Best to split those things up next time.